pkg/users/service.go

```diff
@@ -0,0 +1,40 @@
+package users
+
+import (
+	"fmt"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/logging"
+)
+
+// Service specifies an API to the domain service.
+type Service interface {
```

Comment:
Shouldn't this change be in another commit?

Reply:
I must've done a bad `-i` rebase... lol! done :smile: